Shipr-Helm/templates/redis-service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{  .Values.redis.name }}
+spec:
+  selector:
+    app: {{  .Values.redis.name }}
+    {{- if .Values.redis.extraLabels }}
+    {{- range $key, $value := .Values.redis.extraLabels }}
+    {{ $key }}: {{ $value }}
+    {{- end }}
+    {{- end }}
+  ports:
+    - protocol: TCP
+      port: {{  .Values.redis.service.port }}
+      {{- if or (eq .Values.payment.type "NodePort") (eq .Values.payment.type "LoadBalancer") }}
+      targetPort: {{  .Values.redis.service.port }}
```

Comment:
is targetPort specific to NodePort and LoadBalancer?

Reply:
Fixed